tools/docker/tests/test_flows.sh

```diff
@@ -210,8 +211,11 @@ main() {
 }
 
 VERBOSE=false
-AL_MAC=
-RADIO_WLAN0_MAC=
-RADIO_WLAN2_MAC=
+AL_MAC_agent1=
```

Comment:
These don't need to be initialised. We only initialise default values for variables that can be overridden by options, but these variables are internal variables that will be set later.